core/server/master/src/main/java/alluxio/master/file/options/WorkerHeartbeatOptions.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.master.file.options;
+
+import alluxio.thrift.FileSystemHeartbeatTOptions;
+import alluxio.thrift.UfsFileTStatus;
+import alluxio.underfs.UfsFileStatus;
+
+import com.google.common.base.Objects;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.annotation.concurrent.NotThreadSafe;
+
+/**
+ * Method options for syncing the metadata of a path.
```

Comment:
This should be more general since we may add new options

Reply:
Done.